custom_components/tuya_local/devices/sefaul_q9_dehumidifier.yaml

```diff
@@ -0,0 +1,68 @@
+name: Dehumidifier
+products:
+  - id: e8c9gukirorvyhk8
+    manufacturer: Sefaul
+    model: Q9 Dehumidifier
+
+entities:
+  - entity: light
+    class: light

```

Comment:
There is no class "light", and no need to add a name that is the same as the entity type either, it is better to let it default to the translated default name of the entity.